corehq/apps/reports/util.py

```diff
@@ -11,11 +11,7 @@
 from corehq.apps.users.permissions import get_extra_permissions
 
 from couchexport.util import SerializableFunction
-from couchforms.analytics import (
-    get_all_user_ids_submitted,
-    get_first_form_submission_received,
-    get_username_in_last_form_user_id_submitted,
-)
+from couchforms.analytics import get_first_form_submission_received
```

Comment:
This last one is used in the datespan filter, but it's fixed when someone uses new exports feature flag, so not touching right now